projects/libevent/bufferevent_fuzzer.cc

```diff
@@ -56,30 +59,48 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
     }
     bev1 = pair[0];
     bev2 = pair[1];
-    bufferevent_pair_get_partner(bev1);
+    if (!bufferevent_pair_get_partner(bev1)) {
+      printf("Bufferevent partner is not found\n");
```

Comment:
Will Ci fails if this printf will be triggered? I think assert is OK, I saw discussion in this PR about it may introduce lots of crashes, but this should not fail.

Reply:
@azat  No. Crashes won't be generated for **printf**, but for **assert** will. Do you want to change it only for the specified test case or for other cases as well?